platform/source/mbed_sdk_boot.c

```diff
@@ -87,6 +87,9 @@ void _platform_post_stackheap_init(void)
     us_ticker_init();
 #endif
 }
+// The below resolves the missing variable warning while config with --keep command with os_cb_sections
```

Comment:
Can you rephrase to something like:
"Define an empty os_cb_sections to remove a RTX warning when building with no RTOS due to the `--keep=os_cb_sections` linker option"

Reply:
Maybe also add somewhere there that `os_cb_section` is normally defined in the `rtos` lib?

Reply:
Yes rephrased

Reply:
Yes i have mentioned that in ticket description